webdriver-spec.html

```diff
@@ -4988,10 +4992,23 @@
      <li><p>Let <var>text</var> be the result of
       joining together each element in <var>character array</var> to a string.
 
-     <li><p>Complete implementation specific steps equivalent to setting the
-       <a>selected files</a> on the <a><code>input</code></a>.
+     <li><p>Let <var>files</var> be the result of
+      splitting <var>text</var> on the newline (<code>'\n'</code>)
+      character.
+
+     <li><p>If <var>files</var> is of 0 length, return an <a>error</a>
+      with <a>error code</a> <a>invalid argument</a>.
 
-     <li><p class=issue>Need to handle <code>&lt;input type=file multiple&gt;</code>
+     <li><p>if <var>element</var>'s <a><code>multiple</code> attribute</a>
+      is <code>false</code> and the length of <var>files</var> is
```

Comment:
since `multiple` is a boolean attribute it's attribute won't be false. We need to check if `hasAttribute` results.